doc/how-to-package-haskell-code.rst

```diff
@@ -0,0 +1,291 @@
+How to package Haskell code
+===========================
+
+.. TIP::
+    If this is your first time using `cabal` you should check out the :doc:`Getting Started guide <getting-started>`.
+
+Starting from scratch, we're going to walk you through creating a simple
```

Comment:
This sounds like it _is_ the Getting Started guide.

Reply:
On second thought, given @ffaf1's comment on a deleted line, this whole thing is just relocated and this confusion was already there. We should probably still figure out what we want.

Reply:
Yeah, the idea is that the getting started is a heavily condensed version of the more detailed package guide.